app/src/main/java/net/squanchy/notification/NotificationCreator.kt

```diff
@@ -60,7 +61,7 @@ class NotificationCreator(private val context: Context) {
             .setContentIntent(createPendingIntentForSingleEvent(event.id))
             .setContentTitle(event.title)
             .setColor(getTrackColor(event))
-            .setWhen(event.startTime.toDateTime().millis)
+            .setWhen(event.startTime.atZone(ZoneId.systemDefault()).toInstant().toEpochMilli())
```

Comment:
Now that I see it, we're doing this dance to convert to millis in a few places; what about an extension function `LocalDateTime.toEpochMilli()`?

Reply:
1761646